govc/cluster/override/change.go

```diff
@@ -62,6 +63,16 @@ func (cmd *change) Register(ctx context.Context, f *flag.FlagSet) {
 	cmd.das.DasSettings = new(types.ClusterDasVmSettings)
 
 	f.StringVar((*string)(&cmd.das.DasSettings.RestartPriority), "ha-restart-priority", "", "HA restart priority: "+strings.Join(rp, ", "))
+
+	rc := []string{
+		string(types.ClusterVmReadinessReadyConditionPoweredOn),
+		string(types.ClusterVmReadinessReadyConditionGuestHbStatusGreen),
+		string(types.ClusterVmReadinessReadyConditionAppHbStatusGreen),
+		string(types.ClusterVmReadinessReadyConditionUseClusterDefault),
+	}
+
+	f.StringVar((*string)(&cmd.orc.VmReadiness.ReadyCondition), "ha-readiness", "", "HA VM readiness: "+strings.Join(rc, ", "))
+	f.Var(flags.NewInt32(&cmd.orc.VmReadiness.PostReadyDelay), "ha-restart-delay", "HA additional delay seconds")
```

Comment:
I'd suggest to rename `ha-restart-delay` to `ha-additional-delay` as it matches the wording in vcenter UI